sirepo/auth/__init__.py

```diff
@@ -23,6 +23,7 @@
 import importlib
 import sirepo.auth_role
 import sirepo.feature_config
+import sirepo.simulation_db
```

Comment:
Would it not be better to defer initialization until init_apis? Is there a reason to import sooner?

Reply:
It's still [a circular import](https://github.com/radiasoft/sirepo/blob/d2624a6728449a9217c991016f81eac872e94b2d/sirepo/simulation_db.py#L29) which is a bad thing. I think auth should defer, since simulation_db is more complex and "lower down" the food chain.

Reply:
Urgh, of course. Thanks. 85119cb7